source/webhook-trigger.py

```diff
@@ -71,3 +75,21 @@ def verify_environment(variables):
         if (value is None):
             logging.error(f"Missing variable [{variable}] - {variables[variable]}")
             sys.exit(1);
+
+def sendSlackErrorMessage(event, status_code):
+    payload = extract_payload(event)
```

Comment:
I'd rename payload to `values` as we're only really initialising it with the payload content before potentially appending to it